pkg/test/certs.go

```diff
@@ -15,134 +15,149 @@
 package test
 
 const (
-	// TODO: update the certs before 2022-02-01
+	// TODO: update the certs before 2025-02-01
```

Comment:
As the `-days 365` arguments in commands of README, it should be updated before 2023? Or maybe you should change the `-days` in README?

Reply:
I used `-days 1000` when generating these certificates. I thought 365 might be more common so I used it in the doc, but this inconsistency does introduce confusion. I updated the README to use `-days 1000`. Thanks!